packages/web-app-admin-settings/src/components/Users/SideBar/EditPanel.vue

```diff
@@ -134,12 +145,16 @@ export default defineComponent({
     },
     compareSaveDialogOriginalObject() {
       return cloneDeep({ ...this.user, passwordProfile: { password: '' } })
+    },
+    selectedRoleName() {
+      return this.roles.find((role) => role.id === this.editUser.appRoleAssignments[0].appRoleId)
```

Comment:
Needs to be wrapped by a `this.$gettext()` call.

Reply:
Oppsie